main.go

```diff
@@ -93,7 +94,7 @@ func main() {
 			}
 			tests[t.name] = t
 			currTest = t
-			fmt.Printf("%s %s %s", run(" RUN "), lightGrey(t.packageName), t.name)
+			fmt.Printf("%s %s %s", run(" RUNS "), lightGrey(t.packageName), t.name)
```

Comment:
This is a bit weird but it's what Jest does, I suspect because it's also 4 letters which means it aligns nicely visually with "PASS" and "FAIL"